tests/unit/test_cromshellconfig.py

```diff
@@ -0,0 +1,125 @@
+        self, mock_data_path
+    ):
+        reload(cromshellconfig)
+        test_cromwell_url = "https://cromwell-v1.dsde-methods.broadinstitute.org"
+        mock_workflow_id = "d689adec-c600-4e4b-be37-4e30e65848c7"
+        mock_cromwell_url = "https://cromwell-v45.dsde-methods.broadinstitute.org"
+
+        # Set the submission file path to mock data
+        cromshellconfig.submission_file = os.path.join(
+            mock_data_path, "all.workflow.database.tsv"
+        )
+        # Execute function that changes cromwell server based off the
+        # workflow id and submission file
+        cromshellconfig.resolve_cromwell_config_server_address(
+            workflow_id=mock_workflow_id
+        )
+        assert (
+            cromshellconfig.cromwell_server == mock_cromwell_url
+        ), f"Cromwell server variable should be set to {mock_cromwell_url}"
+
+        # Cromwell url parameter should supersede workflow id look up
+        cromshellconfig.resolve_cromwell_config_server_address(
+            server_user=test_cromwell_url, workflow_id=mock_workflow_id
+        )
+        assert (
+            cromshellconfig.cromwell_server == test_cromwell_url
+        ), f"Cromwell server variable should be set to {test_cromwell_url}"
+
+    def test_config_dir(self):
+        reload(cromshellconfig)
+        assert cromshellconfig.config_dir is not None, "Config variable should be set"
+
+        assert cromshellconfig.config_dir == os.path.join(
+            Path.home(), ".cromshell"
+        ), "Config file variable path should be in .cromshell"
+
+        assert Path(
+            cromshellconfig.config_dir
+        ).exists(), "Config directory should exist "
+
+    def test_submission_file(self):
+        reload(cromshellconfig)
+        assert (
+            cromshellconfig.submission_file is not None
+        ), "Submission file variable should be set "
+
+        path_to_submission = os.path.join(
+            Path.home(), ".cromshell", "all.workflow.database.tsv"
+        )
+        assert (
+            cromshellconfig.submission_file == path_to_submission
+        ), f"Submission file path should be {path_to_submission} "
+
+        assert Path(
+            cromshellconfig.submission_file
+        ).exists(), "Submission file should exist"
+
+    def test_cromwell_server(self):
+        reload(cromshellconfig)
+        assert (
+            cromshellconfig.cromwell_server is not None
+        ), "Cromwell server should be set"
+
+    def test__load_cromshell_config_options(self, mock_data_path):
+        reload(cromshellconfig)
+        cc__load_cromshell_config_file = getattr(
+            cromshellconfig, "__load_cromshell_config_file"
+        )
+        returned_value = cc__load_cromshell_config_file(os.path.dirname(mock_data_path))
+
+        assert returned_value is not None, "Variable should not be empty"
+        assert type(returned_value) is dict, "Variable should be a dictionary"
```

Comment:
I think this test should be expanded:
currently the mock config file has only one line setting the cromwell server, but what about we add more config settings in that file?

Reply:
Add Todo for this

Reply:
done

Reply:
as a Github issue or as a TODO in the code?

Reply:
In the code, Line 126 of cromshellconfig.py